standard/standard-library.md

```diff
@@ -658,3 +652,400 @@ namespace System.Runtime.CompilerServices
     }
 }
 ```
+
+## C.4 Format Specifications
+
+The meaning of the formats, as used in interpolated string expressions (§interpolated-string-expressions), are defined in ISO/IEC 23271:2012. For convenience the following text is copied from the description of `System.IFormatable`.
+
+**This text is informative.**
+
+A *format* is a string that describes the appearance of an object when
+it is converted to a string. Either standard or custom formats can be used. A
+standard format takes the form *Axx*, where *A* is a single
+alphabetic character called the *format specifier*
+, and *xx* is an integer between zero and 99 inclusive, called the *precision specifier*. The format specifier controls the type
```

Comment:
Move the comma to the previous line, here and below? (In general we might want to rewrap this paragraph.)

Reply:
See below…

Reply:
Resolving as we are going to keep just the XML as-is.

Reply:
Unresolving due to an extra space before the comma being incorrect.